app/javascript/controllers/map_controller.js

```diff
@@ -99,38 +99,42 @@ export default class extends Controller {
     )
   }
 
+  stopPropagation(event) {
```

Comment:
Is this being used somewhere?

Reply:
Yes, this is being attached to the tokens: https://github.com/jdbann/table/pull/44/files#diff-943121b3b4845509815fc5be93776194R7

As the tokens were initially being handled with `mousedown`, this was capturing the event and preventing the map from being dragged. Switching the tokens to drag events meant the `mousedown` event still propagated to the map.